packages/flutter/test/material/floating_action_button_theme_test.dart

```diff
@@ -0,0 +1,162 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
```

Comment:
Please add tests for the debugFillProperties method like:
https://github.com/flutter/flutter/blob/master/packages/flutter/test/material/material_test.dart#L61
https://github.com/flutter/flutter/blob/master/packages/flutter/test/material/material_test.dart#L73

Reply:
Done